core/templates/dev/head/pages/load_constants.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var request = new XMLHttpRequest();
+request.overrideMimeType('application/json');
+request.open('GET', '/assets/constants.json', false);
+request.send(null);
+var CONSTANTS = JSON.parse(request.responseText);
```

Comment:
Ok, I will put this lines in IIFE and create individual `oppia.constant()` for every key.